.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+bower_components
+.idea/workspace.xml
```

Comment:
This specific include should be part of the local gitignore. Please remove line 3.